changelog.d/pa-3056.fixed

```diff
@@ -0,0 +1,18 @@
+Programs like
+
+fn f(x:i32) {
+  x;
+  x;
+  x;
+}
+
+
+in expression-oriented languages, such as Rust, will treat the last x
+as if it was "return x".
+
+Currently we use a hack to support this. To support it in a more
```

Comment:
so you need to understand that the changelog is mostly directed to users of Semgrep; At each release they can have a look at the changelog and know what's new. They don't want to learn about how things were before, that something was a hack, etc. so keep it to the point. In some cases because it's really an internal change and the actual visible change for the user will be in another PR you don't have to add a changelog.d/ entry.

For your PR, we do not support yet this implicit return thing (it will be in another PR), so maybbe just say:
```
Fixed the CFG for expression-oriented languages such as Rust so that on the example below, we generate 3 nodes instead of just one.
```
You can have a longer changelog for the actual PR implementing the implicit return.
```

Reply:
+1 just write what the user will see and the details about how we did it belongs to the code and/or PR description.

Reply:
BTW I forgot or I'm missing why it is useful to keep the first two `x;` in the IL?

Reply:
TBH I would personally not add any changelog for this. Right now it won't have any user-visible effect, it's just an internal thing we're doing in preparation for supporting implicit returns.

Reply:
Sorry about that then. But yeah if this will not be noticed by any user then it doesn't need a changelog entry.

Reply:
Alright, I'll remove this file. Thanks!

Reply:
Removed!